sphinx/texinputs/LICRlatin2utf8.xdy

```diff
@@ -0,0 +1,236 @@
+;; style file for xindy
+;; filename: LICRlatin2utf8.xdy
+;; description: style file for xindy which maps back LaTeX Internal
+;;   Character Representation of letters (as arising in .idx index
+;;   file) to UTF-8 encoding for correct sorting by xindy.
+;; usage: for use with the pdflatex engine,
+;;        *not* for use with xelatex or lualatex.
+;;
+;; This is based upon xindy's distributed file tex/inputenc/utf8.xdy.
+;; The modifications include:
+;;
+;; - Updates for compatibility with current LaTeX macro encoding.
+;;
+;; - Systematic usage of the \IeC {...} mark-up, because mark-up in
+;;   tex/inputenc/utf8.xdy was using it on seemingly random basis, and
+;;   Sphinx coercing of xindy usability for both Latin and Cyrillic scripts
+;;   with pdflatex requires its systematic presence here.
+;;
+;; - Support for some extra letters: Ÿ, Ŋ, ŋ, Œ, œ, Ĳ, ĳ, ȷ and ẞ.
+;;
+;;   Indeed Sphinx needs to support for pdflatex engine all Unicode letters
+;;   available in TeX T1 font encoding.  The above letters are found in
+;;   that encoding but not in the Latin1, 2, 3 charsets which are those
+;;   covered by original tex/inputenc/utf8.xdy.
+;;
+;; - There is a problem that ȷ is not supported out-of-the box by LaTeX
+;;   with inputenc, one must add explicitely
+;;       \DeclareUnicodeCharacter{0237}{\j}
+;;   to preamble of LaTeX document.  However this character is not supported
+;;   by the TeX "times" font used by default by Sphinx for pdflatex engine.
+;;
+;; - ẞ needs \DeclareUnicodeCharacter{1E9E}{\SS} (but ß needs no extra set-up).
+;;
+;; - U+02DB (˛) and U+02D9 (˙) are also not supported by inputenc
+;;   out of the box and require
+;;       \DeclareUnicodeCharacter{02DB}{\k{}}
+;;       \DeclareUnicodeCharacter{02D9}{\.{}}
+;;   to be added to preamble.
```

Comment:
MEMO: dev repo of LaTeX contains a commit https://github.com/latex3/latex2e/commit/ed72e7171789b792a07f040d6e5d07f96623aec8 which adds these 4 missing characters to the list of those supported out of the box by pdflatex + utf8 inputenc